presto-tests/src/test/java/io/prestosql/tests/TestDictionaryAggregation.java

```diff
@@ -26,18 +28,19 @@
 public class TestDictionaryAggregation
         extends AbstractTestQueryFramework
 {
-    public TestDictionaryAggregation()
+    @Override
+    protected QueryRunner createQueryRunner()
     {
-        super(() -> {
-            LocalQueryRunner queryRunner = LocalQueryRunner.create(testSessionBuilder()
-                    .setSystemProperty(DICTIONARY_AGGREGATION, "true")
-                    .setSystemProperty(REORDER_JOINS, "false") // no JOIN reordering
-                    .build());
-
-            queryRunner.createCatalog("tpch", new TpchConnectorFactory(1), ImmutableMap.of());
+        return LocalQueryRunner.create(testSessionBuilder()
+                .setSystemProperty(DICTIONARY_AGGREGATION, "true")
+                .setSystemProperty(REORDER_JOINS, "false") // no JOIN reordering
+                .build());
+    }
 
-            return queryRunner;
-        });
+    @BeforeClass
+    public void setUp()
+    {
+        ((LocalQueryRunner) getQueryRunner()).createCatalog("tpch", new TpchConnectorFactory(1), ImmutableMap.of());
```

Comment:
I guess you considered putting this method to `createQueryRunner` like `TestGoogleSheets`. Do you think splitting like this is better than defining with the method? I'm asking this question for the consistency.

Reply:
Done